legacy_packages/sdk/src/evm/zksync/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const SINGLETON_FACTORY = "0xa51baf6a9c0ef5Db8C1898d5aDD92Bf3227d6088";
+export const PUBLISHED_PRIVATE_KEY = "";
```

Comment:
assuming this is supposed to be empty?

Reply:
Yes. Would likely be needed for zksync subchains. Not necessary at the moment.